src/SwipeProfile.css

```diff
@@ -1,5 +1,107 @@
 .SwipeProfile {
-    text-align: center;
-    margin: auto;
-    padding: 20px;
-  }
\ No newline at end of file
+  text-align: center;
```

Comment:
add in font to change the font of "potential friends". Font details are in figma

Reply:
Okk